### ISSUE TITLE
bug: SQS not passing initial health check in version 0.13.0

### ISSUE THREAD

**Issue by samlaycock** [localstack/localstack#4998]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When running `localstack/localstack:0.13.0` on an M1 Mac with `sqs` as an enabled service, any requests that are proxied to SQS fail because the SQS services fails it's initial health check:

```
2021-11-24 08:04:58,055 INFO success: infra entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2021-11-24T08:05:16:INFO:localstack.services.infra: Starting mock SQS service on http port 4566 ...
2021-11-24T08:05:16:INFO:localstack.services.infra: Starting mock SQS service on http port 4566 ...
2021-11-24T08:05:16:ERROR:localstack.services.sqs.sqs_starter: SQS health check failed
Traceback (most recent call last):
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/parsers.py", line 479, in _parse_xml_string_to_dom
    parser.feed(xml_string)
xml.etree.ElementTree.ParseError: syntax error: line 1, column 54

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/code/localstack/localstack/services/sqs/sqs_starter.py", line 46, in check_sqs
    out = aws_stack.connect_to_service(
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/client.py", line 391, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/client.py", line 705, in _make_api_call
    http, parsed_response = self._make_request(
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/client.py", line 725, in _make_request
    return self._endpoint.make_request(operation_model, request_dict)
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/endpoint.py", line 102, in make_request
    return self._send_request(request_dict, operation_model)
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/endpoint.py", line 134, in _send_request
    success_response, exception = self._get_response(
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/endpoint.py", line 165, in _get_response
    success_response, exception = self._do_get_response(
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/endpoint.py", line 216, in _do_get_response
    parsed_response = parser.parse(
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/parsers.py", line 243, in parse
    parsed = self._do_error_parse(response, shape)
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/parsers.py", line 533, in _do_error_parse
    root = self._parse_xml_string_to_dom(xml_contents)
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/parsers.py", line 482, in _parse_xml_string_to_dom
    raise ResponseParserError(
botocore.parsers.ResponseParserError: Unable to parse response (syntax error: line 1, column 54), invalid XML received. Further retries may succeed:
b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>\n'
2021-11-24T08:05:16:ERROR:localstack.services.plugins: error while checking service sqs: 
2021-11-24T08:05:16:WARNING:localstack.utils.server.http2_server: Error in proxy handler for request POST http://localhost:4566/: failed to get service for sqs:  Traceback (most recent call last):
  File "/opt/code/localstack/localstack/services/edge.py", line 220, in _require_service
    self.service_manager.require(api)
  File "/opt/code/localstack/localstack/utils/bootstrap.py", line 91, in wrapped
    return f(*args, **kwargs)
  File "/opt/code/localstack/localstack/services/plugins.py", line 331, in require
    raise container.errors[-1]
  File "/opt/code/localstack/localstack/services/plugins.py", line 239, in check
    self.service.check(print_error=True)
  File "/opt/code/localstack/localstack/services/plugins.py", line 184, in check
    return self.check_function(expect_shutdown=expect_shutdown, print_error=print_error)
  File "/opt/code/localstack/localstack/services/sqs/sqs_starter.py", line 55, in check_sqs
    assert out is not None
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/code/localstack/localstack/utils/server/http2_server.py", line 159, in index
    result = await run_sync(handler, request, data)
  File "/opt/code/localstack/localstack/utils/async_utils.py", line 86, in run_sync
    return await loop.run_in_executor(thread_pool, copy_context().run, func_wrapped)
  File "/opt/code/localstack/localstack/utils/run.py", line 150, in run
    result = self.func(self.params, **kwargs)
  File "/opt/code/localstack/localstack/utils/async_utils.py", line 31, in _run
    return fn(*args, **kwargs)
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 669, in handler
    response = modify_and_forward(
  File "/opt/code/localstack/localstack/utils/aws/request_context.py", line 205, in modify_and_forward
    result = modify_and_forward_orig(
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 338, in modify_and_forward
    listener_result = listener.forward_request(
  File "/opt/code/localstack/localstack/services/edge.py", line 165, in forward_request
    self._require_service(api)
  File "/opt/code/localstack/localstack/services/edge.py", line 222, in _require_service
    raise HTTPErrorResponse("failed to get service for %s: %s" % (api, e), code=500)
localstack.utils.server.http2_server.HTTPErrorResponse: failed to get service for sqs: 

2021-11-24T08:05:16:INFO:localstack.utils.run: Thread run method <function AdaptiveThreadPool.submit.<locals>._run at 0xffffb005daf0>(None) failed: failed to get service for sqs:  Traceback (most recent call last):
  File "/opt/code/localstack/localstack/services/edge.py", line 220, in _require_service
    self.service_manager.require(api)
  File "/opt/code/localstack/localstack/utils/bootstrap.py", line 91, in wrapped
    return f(*args, **kwargs)
  File "/opt/code/localstack/localstack/services/plugins.py", line 331, in require
    raise container.errors[-1]
  File "/opt/code/localstack/localstack/services/plugins.py", line 239, in check
    self.service.check(print_error=True)
  File "/opt/code/localstack/localstack/services/plugins.py", line 184, in check
    return self.check_function(expect_shutdown=expect_shutdown, print_error=print_error)
  File "/opt/code/localstack/localstack/services/sqs/sqs_starter.py", line 55, in check_sqs
    assert out is not None
AssertionError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/code/localstack/localstack/utils/run.py", line 150, in run
    result = self.func(self.params, **kwargs)
  File "/opt/code/localstack/localstack/utils/async_utils.py", line 31, in _run
    return fn(*args, **kwargs)
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 669, in handler
    response = modify_and_forward(
  File "/opt/code/localstack/localstack/utils/aws/request_context.py", line 205, in modify_and_forward
    result = modify_and_forward_orig(
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 338, in modify_and_forward
    listener_result = listener.forward_request(
  File "/opt/code/localstack/localstack/services/edge.py", line 165, in forward_request
    self._require_service(api)
  File "/opt/code/localstack/localstack/services/edge.py", line 222, in _require_service
    raise HTTPErrorResponse("failed to get service for %s: %s" % (api, e), code=500)
localstack.utils.server.http2_server.HTTPErrorResponse: failed to get service for sqs: 
```

### How are you starting LocalStack?

With a docker-compose file

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

```
localstack:
    image: localstack/localstack:0.13.0
    ports:
      - 4566:4566
    environment:
      - SERVICES=dynamodb,kinesis,s3,ses,sns,sqs
      - LS_LOG=trace
```

I'm using the above `docker compose` config, but the problem can be replicated with the `docker run` command below as well.

`docker run -e SERVICES=sqs -p 4566:4566 localstack/localstack:0.13.0`

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

```js
const AWS = require("@aws-sdk/client-sqs");

const sqsClient = new AWS.SQS({
  endpoint: "http://localhost:4566",
  region: "us-east-1",
  credentials: {
    accessKeyId: "access_key",
    secretAccessKey: "secret_key"
  }
});

(async () => {
  await sqsClient.createQueue({ QueueName: "test-queue" });
})();
```

### Environment

```markdown
- OS: MacOS 12.1 (21C5039b) (ARM 64)
- LocalStack: 0.13.0
```

**Comment by dpwdec** [comment]
@samlaycock I'm having the same issue. Did you ever figure out what was causing it and how it might be fixed?

**Comment by thrau** [comment]
This issue should be solved with the new SQS provider in 0.14.3 which has a completely different startup procedure. Please open a new issue if the problem persists.
